ui/src/app/components/spinners.tsx

```diff
@@ -31,7 +31,7 @@ export const Spinner = ({dark = false, size = 72, style = {}, ...props}) => {
   </svg>;
 };
 
-interface SpinnerOverlayProps {
+export interface SpinnerOverlayProps {
```

Comment:
Interesting - does the compiler complain if this is not exported?

Reply:
I'm using it in `with-spinner-overlay`.